src/components/timeline/map.js

```diff
@@ -114,28 +116,24 @@ export function DeploymentMap({
           <DeploymentLayer
             config={{
               id: "static-locations",
-              type: "circle",
+              type: "symbol",
               source: "deployment",
-              paint: {
-                "circle-color": "#E8E845",
-                "circle-opacity": {
-                  base: 1.5,
-                  stops: [
-                    [10, 0.65],
-                    [14, 0.85],
-                  ],
-                },
-                "circle-radius": {
-                  base: 1.5,
-                  stops: [
-                    [10, 4],
-                    [16, 10],
-                    [20, 16],
-                  ],
-                },
-                "circle-stroke-width": 1,
-                "circle-stroke-opacity": 0.1,
-                "circle-stroke-color": "#E8E845",
+              layout: {
+                "icon-image": iconImage,
+                "icon-allow-overlap": true,
+                "icon-size": [
+                  "interpolate",
+                  ["exponential", 0.3],
+                  ["zoom"],
+                  0,
+                  0.1,
+                  12,
+                  0.6,
+                  18,
+                  1,
+                  22,
+                  1.5,
```

Comment:
```suggestion
                  0,
                 0.25,
                  12,
                  0.75,
                  18,
                  1.25,
                  22,
                  1.75
```
@willemarcel proposed change to make icons slightly larger